### PR TITLE
Add avatar for login button in Header.vue

### LINKED ISSUE #625: Change Logged in user style to an avatar icon

### PR DESCRIPTION
**Pull request by hoseinmirian**

### CODE REVIEW

**Review by knirirr — APPROVED** [review]

File: src/components/Navigation/Header.vue [hunk 1]
```diff
@@ -71,9 +77,13 @@
             :small="$vuetify.breakpoint.mdAndDown"
             :x-large="$vuetify.breakpoint.xlOnly"
             class="mr-1 mt-sm-1 teal darken-2"
-            to="/accounts/profile"
```
Comment (terazus) [inline src/components/Navigation/Header.vue:74]: Why did you remove this line? the link to the profile page doesn't work anymore once you are logged in.

File: src/components/Navigation/Header.vue [hunk 2]
```diff
@@ -71,9 +77,13 @@
             :small="$vuetify.breakpoint.mdAndDown"
             :x-large="$vuetify.breakpoint.xlOnly"
             class="mr-1 mt-sm-1 teal darken-2"
```
Comment (terazus) [inline src/components/Navigation/Header.vue]:
```suggestion
            class="mr-1 mt-sm-1 teal darken-2 pl-2"
```
Reduce padding left before the icon to harmonize the button

**Review by terazus — CHANGES_REQUESTED** [review]

File: src/components/Navigation/Header.vue (hunk 1 above)
Comment (hoseinmirian) [reply src/components/Navigation/Header.vue:74]: its been removed accidentally

File: src/components/Navigation/Header.vue (hunk 2 above)
Comment (hoseinmirian) [reply src/components/Navigation/Header.vue]: added

**Comment by hoseinmirian** [comment]
@terazus applied the changes u requested as well as
- moved data block above computed as webstorm insists. 
- moved ellipses class as it can be used generally in the main.scss

**Review by terazus — APPROVED** [review]
